addons/snailmail/static/src/messaging/component/snailmail_error_dialog/snailmail_error_dialog.xml

```diff
@@ -0,0 +1,59 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+
+    <t t-name="snailmail.messaging.component.SnailmailErrorDialog" owl="1">
+        <Dialog contentClass="'o_SnailmailErrorDialog'" title="title" size="'medium'" t-ref="dialog">
```

Comment:
shouldn't it have top-level `t-if` on `message` at least?

Reply:
No idea, moderation dialog don't have it and I copied them.

Reply:
I don't want to waste time on this issue, added `t-if` on `message`.